codewars23.py

```diff
@@ -0,0 +1,26 @@
+# https://www.codewars.com/kata/52f787eb172a8b4ae1000a34/train/python
+
+def zeros(n):
+    if n >= 5:
+        five_power = 1
+    else:
+        five_power = 0
+    x = n // 5
+    while True:  # this loop finds the highest power of 5 in range 1 to n.
+        # I know there must be an easier way to do this.
+        if x // 5 >= 1:
+            five_power += 1
+            x = x // 5
+        else:
+            break
+    y = 0
+    while True:
```

Comment:
I think that if one uses two loops which are the same (lines 9 and 17), it means that already there's something that can be simplified. At least from top of my head, I cannot come up with an example, where it would be useful. With proper if and elif statements, you can make it work within one `while` loop. I mean here the code from lines 9-22.

Reply:
Thanks for spotting this! I agree, making two loops instead of one made no sense. Merged it into one!